src/common/logger/logger.cpp

```diff
@@ -50,15 +54,23 @@ void Logger::init(std::string loggerName, std::wstring logFilePath, std::wstring
     auto logLevel = getLogLevel(logSettingsPath);
     try
     {
-        auto sink = make_shared<sinks::daily_file_sink_mt>(logFilePath, 0, 0, false, LogSettings::retention);
-        logger = make_shared<spdlog::logger>(loggerName, sink);
+        auto sink = make_shared<daily_file_sink_mt>(logFilePath, 0, 0, false, LogSettings::retention);
+        if (IsDebuggerPresent())
+        {
+            auto msvc_sink = make_shared<msvc_sink_mt>();
```

Comment:
SG: use different patterns for different sinks. For msvc_sink it is useful to have the logger name in the log message and process id is redudant. 
`msvc_sink->set_pattern("[%Y-%m-%d %H:%M:%S.%f] [%n] [t-%t] [%l] %v");`

Reply:
good suggestion! done